build/yamls/antrea-prometheus.yml

```diff
@@ -136,7 +136,7 @@ spec:
     spec:
       containers:
         - name: prometheus
-          image: projects.registry.vmware.com/antrea/prom-prometheus:v2.19.3
+          image: projects.registry.vmware.com/antrea/prom-prometheus:v2.44.0
```

Comment:
@tnqn I verified image prom/prometheus:v2.44.0, it works fine. But I didn't get the latest image from `projects.registry.vmware.com/antrea`, Could you help to upload? Thanks.

Reply:
looks like the image still doesn't exist?

Reply:
yeah, it's not uploaded yet, we may include this change in release v1.14.

Reply:
I think we could use a public image instead of keeping adding new copies. I see at least ubuntu/prometheus is free to use: https://hub.docker.com/r/ubuntu/prometheus
> Current prometheus Docker Image [from Canonical](https://ubuntu.com/security/docker-images), based on Ubuntu. Receives security updates and rolls to newer prometheus or Ubuntu release. **This repository is free to use and exempted from per-user rate limits.**

Reply:
@antoninbas any objection to use the `ubuntu/prometheus` image? The latest version is `v2.46.0-22.04_stable`, it should be good for us.

Reply:
No issue from my side